apps/antalmanac/src/components/Map/Map.tsx

```diff
@@ -177,11 +180,7 @@ export default function CourseMap() {
                 <Paper sx={{ zIndex: 400, position: 'relative', my: 2, mx: 6.942, marginX: '15%', marginY: 8 }}>
                     <Tabs value={selectedDayIndex} onChange={handleChange} variant="fullWidth" sx={{ minHeight: 0 }}>
                         {days.map((day) => (
-                            <Tab
-                                key={day}
-                                label={day || 'All'}
-                                sx={{ padding: 1, minHeight: 'auto', minWidth: '10%', p: 1 }}
-                            />
+                            <Tab key={day} label={day} sx={{ padding: 1, minHeight: 'auto', minWidth: '10%', p: 1 }} />
```

Comment:
Revert this. Also you can remove either "p" or "padding", they're the same thing.

```suggestion
                            <Tab key={day} label={day || 'All'} sx={{ padding: 1, minHeight: 'auto', minWidth: '10%'  }} />
```

Reply:
Fixed